tests/testthat/test-carehomes-vaccination.R

```diff
@@ -274,6 +276,56 @@ test_that("Vaccination of asymptomatic infectious individuals works", {
 })
 
 
+test_that("Vaccination of presymptomatic infectious individuals works", {
+  skip("TODO: can't vaccinate fast enough")
```

Comment:
was this moved from somewhere? We shouldn't add new tests that don't run (we should find time to fix the existing skipped ones at some point)

Reply:
I copied and adapted the other vaccination tests. I think there's 12 total now that are being skipped. Yes, we should fix them.